package.json

```diff
@@ -36,7 +36,8 @@
     "typescript": "^3.7.5",
     "unzipper": "^0.10.11",
     "loglevel": "^1.6.8",
-    "snyk": "^1.316.1"
+    "snyk": "^1.316.1",
```

Comment:
run `npm audit fix` to get the latest packages. It avoids those immediate snyk PRs